docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+web:
+  build: .
+  command: 'bash -c ''npm start'''
+  working_dir: /app/user
+  environment:
+    PORT: 8080
+  ports:
+    - '8080:8080'
+  volumes:
+    - '.:/app/user'
```

Comment:
If node_modules is not present on developer's machine, it will break, even though the Dockerfile does an npm install. We should look into documenting this fact. Maybe create a common issues section in the readme?